src/controllers/dashboard/networking.js

```diff
@@ -136,7 +141,7 @@ define(['loading', 'libraryMenu', 'globalize', 'emby-checkbox', 'emby-select'],
         });
         view.querySelector('#btnSelectCertPath').addEventListener('click', function () {
             require(['directorybrowser'], function (directoryBrowser) {
-                var picker = new directoryBrowser.default();
+                const picker = new directoryBrowser();
```

Comment:
Just curious, was `.default()` changed in another pull request or was it never required here?

Reply:
Thanks for noticing that, I forgot to remove a require statement above. I think I was there in a previous PR? I'm not too sure. Either way, searching for a SSL cetificate is currently broken so I'll make a new PR to fix it